machina/templates/machina/forum_member/forum_profile_update.html

```diff
@@ -12,7 +12,7 @@
                     <h3 class="panel-title">{% trans "Edit forum profile" %}</h3>
                 </div>
                 <div class="panel-body">
-                    <form method="post" action="." class="form" enctype="multipart/form-data" novalidate>{% csrf_token %}
+                    <form method="post" action="." class="form" enctype="multipart/form-data" novalidate>{% csrf_token %}{% if AVATARS_ENABLED %}
```

Comment:
I don't think this is the right way to disable the avatar field in the form. Technically a user could still upload an avatar since the Django form will still accept it. IMHO a better solution would be to manually remove the `avatar` form field in the `FormProfileForm.__init__` method (here: https://github.com/ellmetha/django-machina/blob/master/machina/apps/forum_member/forms.py#L13) if the `MACHINA_PROFILE_AVATARS_ENABLED` setting is set to `False`. 😉